### PR TITLE
feat: Adding project info command to find where project is located

### PR DESCRIPTION
**Pull request by teddylear**
feat: Adding project info command to find where project is located

Implement part of #2038. Only question is testing this change. I attempted to add basic test but nothing concrete. Any ideas to better test this are appreciated. Also I wouldn't mind adding a 'Test' section to the contributing page to let other users understand this better as well with it looking like a non-standard golang testing structure.

### CODE REVIEW

File: tests/project.bats [hunk 1]
```diff
@@ -27,4 +27,8 @@ setup() {
 	assert_output --partial "generated by dagger"
 
 	test ! -f ./cue.mod/pkg/.gitignore
+
+
+	"$DAGGER" project info
+	assert_output --partial "Current dagger project in"
```
Comment (aluzzardi) [inline tests/project.bats:33]: Perhaps you could also check that `"$TEMPDIR"` is part of the output?

**Review by aluzzardi — COMMENTED** [review]
Looks great, thanks for the contribution!

Regarding your question about test -- perhaps all it's missing is checking the project path is part of the output, and then we're good to go

File: tests/project.bats (hunk 1 above)
Comment (teddylear) [reply tests/project.bats:33]: Added! Also addressed linting issues. If there is anything else I should change please let me know.

**Review by aluzzardi — APPROVED** [review]

**Comment by teddylear** [comment]
@aluzzardi I think I fixed issue with integration test, but if it fails again I'm not sure why as it does show the substring.

File: tests/project.bats [hunk 2]
```diff
@@ -27,4 +27,8 @@ setup() {
 	assert_output --partial "generated by dagger"
 
 	test ! -f ./cue.mod/pkg/.gitignore
+
+	"$DAGGER" project info
```
Comment (aluzzardi) [inline tests/project.bats:31]: You will have to add `run ` before `"$DAGGER`, e.g.

```
run "$DAGGER" project info
assert_success
assert_output ...
```

`assert_*` functions are helpers that work with `run` (it's asserting the last run)

Comment (teddylear) [reply tests/project.bats]: Ahh ok, updated with change.

File: cmd/dagger/cmd/project/info.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		lg := logger.New()
+
+		cueModPath, cueModExists := pkg.GetCueModParent()
+		if !cueModExists {
+			lg.Fatal().Msg("dagger project not found. Run `dagger project init`")
+		}
+
+		fmt.Printf("\nCurrent dagger project in: %s", cueModPath)
```
Comment (TomChv) [inline cmd/dagger/cmd/project/info.go:31]: Why no `\n` at the end?

**Review by TomChv — COMMENTED** [review]
Thanks for your contribution, I left a comment.

Since it's a new CLI feature, @shykes should review it too.

File: tests/project.bats [hunk 4]
```diff
@@ -2,11 +2,11 @@ setup() {
 	load 'helpers'
 
 	common_setup
-	
+
 	TEMPDIR=$(mktemp -d)
 }
 
-@test "project init and update" {
+@test "project init and update and info" {
```
[the remark this answers is not in the document]
Comment (teddylear) [reply tests/project.bats:10]: Added negative test

File: cmd/dagger/cmd/project/info.go (hunk 3 above)
Comment (teddylear) [reply cmd/dagger/cmd/project/info.go]: Added.

**Comment by teddylear** [comment]
@TomChv @shykes Updated from feedback. Please let me know if I should change anything else.

**Review by helderco — APPROVED** [review]

**Pull request merged**